README.md

```diff
@@ -76,7 +76,7 @@ The following attributes are available for `pins`:
 
 | Name | Type | Required? | Description |
 | ---- | ---- | --------- | ----------- |
-|`pin`| string | **Required** | The pin number of the board's GPIO pin that you wish to configure the digital interrupt for. |
+|`pin`| string | **Required** | The physical pin number of the board's GPIO pin that you wish to configure the digital interrupt for. |
```

Comment:
would it be worth providing a graphic or some more clarifying explanations at the bottom?